homeassistant/components/climate/eq3btsmart.py

```diff
@@ -81,24 +102,41 @@ def set_temperature(self, **kwargs):
         self._thermostat.target_temperature = temperature
 
     @property
-    def device_state_attributes(self):
-        """Return the device specific state attributes."""
-        return {
-            ATTR_MODE: self._thermostat.mode,
-            ATTR_MODE_READABLE: self._thermostat.mode_readable,
-        }
```

Comment:
Reporting the mode as attribute is quite pointless now that we represent it as device state but there are some values the updated eq3btsmart which should be reported as attributes:
eq3.valve_state (homematic.py reports it as 'Valve'), eq3.locked (nest reports that via a binary sensor as 'is_locked'), eq3.window_open and eq3.low_battery.